canonical-web-and-design/maas-ui
# fix(legacy): QA fixes for subnet boot architectures form

## Done

- In the subnet details edit boot architecture form:
  - Made entire arch row clickable.
  - Made boot arch table sortable.
  - Added link to boot arch discourse page.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to a subnet details page and click "Take action" > "Edit boot architectures"
- Check that there is a "More info" link that takes you to the boot architectures discourse page
- Check that clicking anywhere on a row will toggle the checkbox
- Check that you can sort the table by the table headers

## Fixes

Fixes canonical-web-and-design/MAAS-Squad#2420
